### ISSUE TITLE
PrimitivePDBReader clips character from header lines

### ISSUE THREAD

**Issue by ivirshup** [MDAnalysis/mdanalysis#332]
`PrimativePDBReader` clips the last character from a number of entries in the header by reading those lines as `[6:-1]` when they should be read as`[6:]`.

In addition, there are also a number of header entries which are just ignored during the reading of the file. These entries include `AUTHOR`, `DBREF`, `EXPDTA`, `JRNL`, `KEYWDS`, `NUMMDL`, `REVDAT`, `SEQRES`, `SHEET`, `SOURCE`, and `USER`.

**Comment by orbeckst** [comment]
This is going to be fixed in 0.11 (I am just checking that my fix passes all tests). But the PrimitivePDBReader only gives you

* HEADER
* TITLE (just added)
* COMPND
* REMARK

records.

You can alternatively try the PDBReader (Biopython.PDB) and access the (parsed) header data as 
```python
u = Universe(PDB)
headerdict = u.trajectory.pdb.header
```
or from 0.11 onwards (commit 6558ca25b2c378d3c0b9ea8c8d3db766bdf35a1e):

```python
u = Universe(PDB)
headerdict = u.trajectory.metadata
```
The Bio.PDB header is incomplete, though. The best thing to do is to interactively explore the dictionary and see if it contains what you need.


Unfortunately, there's no uniform interface to metadata yet. Feel free to raise specific issues and in particularly, contribute patches/pull requests. Getting metadata hasn't be highest priority in the past but it wouldn't be hard to add this to the PDB topology parser (instead of the coordinate reader). Let us know what the desired format/data structure should be and raise a new issue with details.

As an example/test case please use PDB ID 4E43, 
```python
import MDAnalysis
from MDAnalysis.tests.datafiles import PDB_full
u = MDAnalysis.Universe(PDB_full)
```

**Comment by orbeckst** [comment]
@ivirshup thanks for the bug report. The next release 0.11.0 will have the fix or you can just pull from the development branch.

By the way, I implemented the proper (PDB standard for the title section](http://www.wwpdb.org/documentation/file-format-content/format33/sect2.html) and the data are read exactly from the prescribed column, so for instance, HEADER is read from column 11 to 66 (i.e. as `line[10:66]`).
